getgather/mcp/main.py

```diff
@@ -107,7 +107,7 @@ async def on_call_tool(self, context: MiddlewareContext[Any], call_next: CallNex
 CATEGORY_BUNDLES: dict[str, list[str]] = {
     "food": ["doordash", "ubereats"],
     "books": [],
-    "shopping": ["amazon", "astro"],
+    "shopping": ["astro"],
```

Comment:
<img width="379" height="343" alt="Image" src="https://github.com/user-attachments/assets/94456391-4287-41bf-b249-aa359f2c8135" />

I think We also need to change GETHER_URL on our demo app.